constants.py

```diff
@@ -46,8 +46,11 @@
 # Select the Model ID and model_basename
 # load the LLM for generating Natural Language responses
 
-MODEL_ID = "TheBloke/Llama-2-7B-Chat-GGML"
-MODEL_BASENAME = "llama-2-7b-chat.ggmlv3.q4_0.bin"
+# MODEL_ID = "TheBloke/Llama-2-7B-Chat-GGML"
+# MODEL_BASENAME = "llama-2-7b-chat.ggmlv3.q8_0.bin"
+
+MODEL_ID = "TheBloke/vicuna-7B-v1.5-GGML"
+MODEL_BASENAME = "vicuna-7b-v1.5.ggmlv3.q8_0.bin"
```

Comment:
Here you added a `GGML` model after removing the same type of model. Not required.